### ISSUE TITLE
after installed dojo/cli, and running dojo command, the programe print "Invalid response from npm search"

### ISSUE THREAD

**Issue by LaiHuanMin** [dojo/meta#263]
Hello, I'm following your website (https://dojo.io/tutorials/001_static_content/), and used npm to install @dojo/cli and @dojo/cli-create-app, the execute command like this : 
```shell
#!/bin/bash
npm install -g @dojo/cli  
npm install -g @dojo/cli-create-app
dojo create app --name first-dojo-app   # right here, the program noticed npm install error, and just exit, the error msg notice because dojo-interface was deprecated, so install is not totally finish, so i use yarn to install
cd first-dojo-app
yarn # instead of not full npm install
dojo build -m dev -w memory -s  
# after execute this command, the dojo print "Invalid response from npm search" info, but i don't know where the invalid response is, and what the invalid content is
```

This is my system info and npm,node env version: 
1, node v8.4.0 
2, npm v5.3.0 
3, Ubuntu 16.04.2 LTS
4, yarn v1.1.0

Is that system environment not satisfield? Should I need to release version? Or change the npm source url?    

I wanna to use dojo, and wanna study by dojo website, but cannot success create app step by step, could you help, Thanks a lot

**Comment by nicknisi** [comment]
Can you provide the error you are seeing?

**Comment by nicknisi** [comment]
Hmm, I'm not having issues running the `dojo` command on Ubuntu. From the looks of it you have your registry set to `https://registry.npm.taobao.org/`. Could that be the issue?

**Comment by LaiHuanMin** [comment]
@nicknisi it's maybe because the registry set question, it will print 'Invalid respon..." error msg, but seems like the error msg impact nothing.

**Comment by nicknisi** [comment]
Yeah, it should still work. @JamesMilnerUK was also experiencing this issue. We'll continue to look into it and find a fix.
